code/game/objects/items/storage/uplink_kits.dm

```diff
@@ -166,7 +166,7 @@
 			new /obj/item/toy/plush/carpplushie/dehy_carp(src)
 			new /obj/item/slimepotion/slime/sentience(src)
 
-		if("mad_scientist")
+		if("mad_scientist") //who the fuck knows
```

Comment:
If we're going by the old numbers there, I'd say 29 TC?

Reply:
I'll add that, as well as a comment explaining my hesitance.  The clusters are not a clean 10 tc each, because they're random which means they could very an immense amount.  And then there are the two energy guns which are unique and really tough to price.

If somebody wants to ponder what the prices of those *should* be and note the price based on that, maybe even make them available to normal traitors, they're welcome!  but I think the special kits are kind of inherently unpriceable to some degree.